src/main/java/mekanism/common/block/BlockBounding.java

```diff
@@ -76,20 +86,82 @@ public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World wo
 			IBlockState state1 = world.getBlockState(tileEntity.mainPos);
 			return state1.getBlock().getPickBlock(state1, target, world, tileEntity.mainPos, player);
 		} catch(Exception e) {
+			e.printStackTrace();
 			return ItemStack.EMPTY;
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 * Delegate to main {@link Block#getDrops(NonNullList, IBlockAccess, BlockPos, IBlockState, int)}.
+	 */
 	@Override
-	public boolean removedByPlayer(IBlockState state, World world, BlockPos pos, EntityPlayer player, boolean willHarvest)
+	public void getDrops(NonNullList<ItemStack> drops, IBlockAccess world, BlockPos pos,IBlockState state, int fortune)
 	{
 		try {
 			TileEntityBoundingBlock tileEntity = (TileEntityBoundingBlock)world.getTileEntity(pos);
-			IBlockState state1 = world.getBlockState(tileEntity.mainPos);
-			return state1.getBlock().removedByPlayer(state1, world, tileEntity.mainPos, player, willHarvest);
+			BlockPos mainPos = tileEntity.mainPos;
+			IBlockState state1 = world.getBlockState(mainPos);
+			state1.getBlock().getDrops(drops, world, mainPos, state1, fortune);
 		} catch(Exception e) {
-			return false;
+			e.printStackTrace();
```

Comment:
log it properly or rewrite it so it doesn't need the try catch (the better option)

Reply:
What are we trying to catch anyway? If it's just a ```NullPointerException```, this can be simply replaced with a null check.

Reply:
That's my best guess as to what it was for, so yeah, just null check it